diff --git a/luci-app-passwall/root/usr/share/passwall/0_default_config b/luci-app-passwall/root/usr/share/passwall/0_default_config
index dbf6d21..dcc1144 100644
--- a/luci-app-passwall/root/usr/share/passwall/0_default_config
+++ b/luci-app-passwall/root/usr/share/passwall/0_default_config
@@ -72,10 +72,7 @@ config global_rules
 	option geosite_url 'https://github.com/Loyalsoldier/v2ray-rules-dat/releases/latest/download/geosite.dat'
 
 config global_app
-	option geoview_file '/usr/bin/geoview'
 	option sing_box_file '/usr/bin/sing-box'
-	option xray_file '/usr/bin/xray'
-	option hysteria_file '/usr/bin/hysteria'
 
 config global_subscribe
 	option filter_keyword_mode '1'
