diff --git a/sing-box/Makefile b/sing-box/Makefile
index 9d6fd4a..f6c4b0e 100644
--- a/sing-box/Makefile
+++ b/sing-box/Makefile
@@ -5,12 +5,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=sing-box
-PKG_VERSION:=1.14.2
+PKG_VERSION:=1.12.23
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/SagerNet/sing-box/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=67dd8f8c37ecaaadcfcafad1f0827eed4b034c963b86fd3aa5c0d7a36876845d
+PKG_HASH:=58b41bd8d6255c2ddc2ae5eace53171e550c9debaa8e16d89871299055b58b1a
 
 PKG_LICENSE:=GPL-3.0-or-later
 PKG_LICENSE_FILES:=LICENSE
